test/DiceTests.cpp

```diff
@@ -0,0 +1,13 @@
+#include <gtest/gtest.h>
+#include "Dice.hpp"
+
+struct DiceTest : public ::testing::Test
+{
+    Dice dice;
+};
+
+TEST_F(DiceTest, check_if_throwIt_returns_number_from_the_range_from_1_to_12)
+{
+    int throw_ = dice.throwIt();
+    ASSERT_TRUE(throw_ <= 12 && throw_ >= 1);
```

Comment:
You can also consider changing this line to those two:
```suggestion
    ASSERT_LE(throw_, 12);
    ASSERT_GE(throw_, 1);
```

BTW Why throw_ with underscore here?

Reply:
"throw" as a keyword is not accepted by the compiler

Reply:
Yea, that's right. throw is a keyword. What about `roll`? Or `diceRoll`? Or `diceThrow`?